libvisual/libvisual/lv_cpu.c

```diff
@@ -133,7 +133,11 @@ static void sigfpe_handler_sse (int signal, struct sigcontext sc)
 LONG CALLBACK win32_sig_handler_sse (EXCEPTION_POINTERS* ep)
 {
 	if (ep->ExceptionRecord->ExceptionCode == EXCEPTION_ILLEGAL_INSTRUCTION) {
+#ifdef _AMD64_
```

Comment:
I have grepped the code base for `_AMD64_` now and did not find any other use. To blend in, we should probably give `VISUAL_OS_WIN32` a sibling `VISUAL_OS_WIN64` and check for that, instead? The fact that I don't find `_AMD64_` on https://sourceforge.net/p/predef/wiki/Architectures/ and that I ran into https://stackoverflow.com/a/17761728/11626624 makes me also wonder if `_AMD64_` the best choice.

Reply:
I mainly found `_AMD64_` within windows' headers as the guards for if `RIP` is present in the struct, so I'm assuming it's mainly an internal define. I could try to add `VISUAL_OS_WIN64` since it might also be helpful to distinguish 64-bit too

Reply:
PS: If 64bit detection in CMake to build `VISUAL_OS_WIN64` turns out hard, I would vote for `_WIN64` over `_AMD64_`: it seems more robust / official to me (and the surrounding code is Windows-only already).

Reply:
It seems easy enough to do
```cmake
...
ELSEIF(WIN32)
  SET(VISUAL_OS_WIN32 yes)
  SET(VISUAL_WITH_MINGW ${MINGW})
  SET(VISUAL_WITH_CYGWIN ${CYGWIN})
  IF(VISUAL_WITH_CYGWIN)
    SET(VISUAL_OS_POSIX yes)
  ENDIF()
  IF(${CMAKE_SIZEOF_VOID_P} EQUAL 8)
    SET(VISUAL_OS_WIN64 yes)
  ENDIF()
ELSE()
...
```

Reply:
Good idea